LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 TC39 members
```

Comment:
@codehag who should have the licence for that?

Reply:
Not sure, @ljharb @littledan what do you think?

Reply:
that's a good question; generally the license needs to come from the contributors, so really i think it'd just be "contributors", and then whoever shows up in the commit log would be included?

Alternatively, it could probably be "ecma and TC39 members". It might be good to get a legal perspective.

Reply:
@xtuc would you be able to make this update sometime in the next day? would be great to have it!

Reply:
What about "TC39 members and other contributors"? I don't really involves ecma?

Reply:
sounds good

Reply:
I'm confused, what should I write?

Reply:
Is this wording odd if Ecma has 'members' and TC39 has 'delegates' though?

FWIW, it looks like test262 has a [BSD license](https://github.com/tc39/test262/blob/master/LICENSE) to `Ecma International` and an [open discussion](https://github.com/tc39/test262/pull/851) in which a concern about it has been escalated to Ecma. Perhaps this should be escalated too?

Reply:
For some formality, how about "Ecma's TC39 Royalty Free Task Group"? 

https://www.ecma-international.org/memento/TC39-RF-TG.htm

It can't go wrong with this.